ksqldb-functional-tests/src/test/resources/rest-query-validation-tests/insert-values.json

```diff
@@ -439,6 +439,8 @@
       },
       "statements": [
         "CREATE STREAM input (id INT KEY, val STRING) WITH (kafka_topic='test_topic', value_format='JSON');",
+        "INSERT INTO input (id VALUES (5);",
```

Comment:
I think these changes are a misunderstanding of one of my previous comments -- my suggestion was to add more columns to the single row being inserted, not to add more rows.